src/fidesops/ops/schemas/saas/saas_config.py

```diff
@@ -26,6 +26,7 @@ class ParamValue(BaseModel):
     identity: Optional[str]
     references: Optional[List[FidesopsDatasetReference]]
     connector_param: Optional[str]
+    flatten: Optional[bool] = False
```

Comment:
I think unpack is a better name, I agree with your reasoning that we need to describe what is happening to each individual record instead of the entire input list.